test/land/test_imsproc.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+################################################
+YY=2021
```

Comment:
I am not sure if this should use YYYY rather than YY. Based on my understanding, YY is the last number of YYYY. For the reader, it is easily confused. In particular line 37 where uses yyyymmddhh.

Reply:
Besides this point, everything others look fine with me. This can be decided if YY is fine by the developer.

Reply:
Thanks @YoulongXia-NOAA for your comments. We use `YY` here to keep consistent with our previous PR #207.

Reply:
Okay, look good with me.